invenio_cli/cli/packages.py

```diff
@@ -67,11 +67,26 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages()
+
+    on_fail = "Some of the packages need to be updated."
+    on_success = "All packages are up to date."
+
+    run_steps(steps, on_fail, on_success)
 
 
 @packages.command()
+@click.argument("packages", nargs=-1, type=str)
```

Comment:
Is this optional? or is it forcing at least one? Does argument allow for defaults? (i.e. [])

Reply:
This can be controlled with the nargs parameter. If it is set to -1, then an unlimited number of arguments is accepted.

The value is then passed as a tuple. Note that only one argument can be set to nargs=-1, as it will eat up all arguments.

As far as I understand, it's optional, and testing it seems to work as intended.

Reply:
Sounds good :) I have made a tiny tiny change to avoid a few lines of code and argument passing :)